pkgs/applications/audio/gradio/default.nix

```diff
@@ -51,15 +49,18 @@ in stdenv.mkDerivation rec {
 
     wrapGAppsHook
     desktop-file-utils
-    gsettings-desktop-schemas
   ] ++ gst_plugins;
 
-  enableParallelBuilding = true;
+  patchPhase = ''
+    # Disable the custom build script to run it in post-install phase
+    sed -i "/meson.add_install_script('meson_post_install.sh')/d" ./meson.build
```

Comment:
Why not use the patch that is already there?

Reply:
Mainly - I try to avoid using patches.
Also because it can be seen and controlled at sight what code does.
Also because `sed` can process the lines if the upstream file if it changed, and changing `sed` seems easier, well seems not in this case of single-line, but overall.

Reply:
One disadvantage of sed is that is easily missed when it no longer applies. Maybe add `grep … &&`?

Also, use `postPatch`, since overriding `patchPhase` will make it impossible to add new patches.

Reply:
Ok.
I am making notes.
So seems the process of patching is still not in consensus.

Reply:
Done.
Reverted the change.